### DISCUSSION TITLE
Lean 3 Checkpoints

### DISCUSSION THREAD

**Discussion by sean-lamont** [lean-dojo/ReProver#52] [Q&A] [answered]
Hi, 

Really appreciate the work on this project. I wanted to ask if it's possible to get the final trained models for Lean 3 to replicate the results in the paper? Similar to the lightning checkpoints for the Lean 4 results found in https://huggingface.co/kaiyuy/leandojo-pl-ckpts

Thanks for the help!

**Comment by yangky11** [comment, marked as answer]
They were in the same repo and you can still find them from the repo's history.
